src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceStatus.kt

```diff
@@ -30,5 +30,28 @@ enum class LicenceStatus {
         RECALLED -> LicenceEventType.RECALLED
       }
     }
+
+    fun searchRelevantLicenceStatus(): List<LicenceStatus> {
+      return listOf(
+        ACTIVE,
+        IN_PROGRESS,
+        SUBMITTED,
+        APPROVED,
+        VARIATION_IN_PROGRESS,
+        VARIATION_SUBMITTED,
+        VARIATION_APPROVED,
+        VARIATION_REJECTED,
+      )
+    }
+
+    fun searchOnProbationLicenceStatus(): List<LicenceStatus> {
```

Comment:
could maybe have a field here = 
```kt
val PROBATION_VALUES = setOf(...
```

and then a helper function on enum of `isWithProbation`?
```kt
fun isWithProbation() = PROBATION_VALUES.contains(this)
```

which would simplify the logic where you call it?

Reply:
Done